libraries/botbuilder-core/src/botState.ts

```diff
@@ -165,12 +160,12 @@ export class BotState implements PropertyManager {
      * This example shows how to synchronously get an already loaded and cached state object:
      *
      * ```JavaScript
-     * const state = await botState.get(context);
+     * const state = botState.get(context);
      * ```
      * @param context Context for current turn of conversation with the user.
      */
     public get(context: TurnContext): any|undefined {
-        const cached: any = context.turnState.get(this.stateKey) as CachedBotState;
+        const cached: CachedBotState = context.turnState.get(this.stateKey);
```

Comment:
code change

Reply:
same

---
In reply to: [219688307](https://github.com/Microsoft/botbuilder-js/pull/477#discussion_r219688307) [](ancestors = 219688307)